h/services/groupfinder.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from zope.interface import implementer
+
+from memex.interfaces import IGroupService
+
+from h import models
+from h.groups.util import WorldGroup
+
+
+@implementer(IGroupService)
+class GroupfinderService(object):
```

Comment:
It might be worth adding a note here to say something like "Ideally this would be called the GroupService to match nomenclature in memex. FIXME: rename/split existing GroupService and rename this."